src/threading.c

```diff
@@ -960,6 +960,32 @@ JL_DLLEXPORT int jl_alignment(size_t sz)
     return jl_gc_alignment(sz);
 }
 
+JL_DLLEXPORT int jl_getaffinity(int16_t tid, char *mask, int cpumasksize) {
+    int nthreads = jl_atomic_load_acquire(&jl_n_threads);
+    if (tid < 0 || tid >= nthreads)
+        jl_error("invalid tid");
+
+    // TODO: relaxed or acquire here?
```

Comment:
`relaxed` is correct iirc, since the acquire above establishes that jl_all_tls_states contains the tid we are looking for.